tests/python/frontend/sklearn/test_forward.py

```diff
@@ -129,24 +135,75 @@ def test_column_transfomer():
+#     st_helper = SklearnTestHelper()
+#     roe = RobustOrdinalEncoder()
+#     data = np.array([[0,1],[0,4],[1,2],[1,10]], dtype=np.float32)
+#     roe.fit(data)
+#     dshape = (relay.Any(), len(data[0]))
+#     _test_model_impl(st_helper, roe, dshape, data)
+
+def test_na_label_encoder():
+    st_helper = SklearnTestHelper()
+    nle = NALabelEncoder()
+    i_put = np.array([[1,2,2,6]], dtype=np.float32)
+    nle.fit(i_put)
+    data = np.array([[np.nan,0,1,2,6]], dtype=np.float32)
+    dshape = (relay.Any(), len(data))
+    _test_model_impl(st_helper, nle, dshape, data)
+
+def test_standard_scaler():
     st_helper = SklearnTestHelper()
-    rle = RobustLabelEncoder()
+    ss = StandardScaler()
+    data = np.array([[0, 0], [0, 0], [1, 1], [1, 1]], dtype=np.float32)
+    ss.fit(data)
+    dshape = (relay.Any(), len(data[0]))
+    _test_model_impl(st_helper, ss, dshape, data)
 
-    data = np.array([1, 2, 3, 4, 5], dtype=np.int32)
-    rle.fit(data)
+def test_kbins_discretizer():
+    st_helper = SklearnTestHelper()
+    kd = KBinsDiscretizer(n_bins=3, encode='ordinal', strategy='uniform')
+    data = np.array([[-2, 1, -4,   -1],
+                    [-1, 2, -3, -0.5],
+                    [ 0, 3, -2,  0.5],
+                    [ 1, 4, -1,    2]], dtype=np.float32)
+    kd.fit(data)
+    dshape = (relay.Any(), len(data[0]))
+    _test_model_impl(st_helper, kd, dshape, data)
+
+# def test_tfidf_vectorizer():
+#     st_helper = SklearnTestHelper()
+#     tiv = TfidfVectorizer()
+#     data = [
+#         'This is the first document.',
+#         'This document is the second document.',
+#         'And this is the third one.',
+#         'Is this the first document?',
+#     ]
     
-    dshape = (len(data),)
-    st_helper.compile(rle, dshape, 'int32', 'inverse_transform')
-    tvm_out = st_helper.run(data)
-    # identity transformation, because of the string input, the actually encoding happens outside 
-    # of tvm in runtime as post processing
-    tvm.testing.assert_allclose(data, tvm_out, rtol=1e-5, atol=1e-5)
+#     dshape = (relay.Any(), len(data))
+#     st_helper.compile(tiv, dshape, 'int32')
+#     sklearn_out = tiv.fit_transform(data).toarray()
+#     tvm_out = st_helper.run(data)
+#     tvm.testing.assert_allclose(sklearn_out, tvm_out, rtol=1e-5, atol=1e-5)
+
+def test_pca():
+    st_helper = SklearnTestHelper()
+    pca = PCA(n_components=2)
+    data = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]], dtype=np.float32)
+    pca.fit(data)
+    dshape = (relay.Any(), len(data[0]))
+    _test_model_impl(st_helper, pca, dshape, data)
 
 if __name__ == "__main__":
     test_simple_imputer()
     test_robust_imputer()
     test_robust_scaler()
     test_column_transfomer()
     test_threshold_onehot_encoder()
-    test_inverse_label_transformer()
+    # test_robust_ordinal_encoder()
+    test_na_label_encoder()
+    test_standard_scaler()
+    test_kbins_discretizer()
+    # test_tfidf_vectorizer()
+    test_pca()
```

Comment:
We dont need this for testing

Reply:
This is the entrance for local testing. This is following the format from other frameworks.

Reply:
understood, thanks